teste2e/common/k8s.py

```diff
@@ -48,25 +48,41 @@ def __init__(self):
         self.operator_pod_name = run_cmd_text(
             "kubectl get pods | grep mizar-operator | awk '{print $1}'")
 
-    def create_vpc(self, name, ip, prefix, dividers=1, vni=None):
-        self.api.create_vpc(name, ip, prefix, dividers, vni)
+    def create_vpc(self, name, ip, prefix, dividers=1):
+        self.api.create_vpc(name, ip, prefix, dividers)
 
-    def create_net(self, name, ip, prefix, vpc, bouncers=1, vni=None):
-        self.api.create_net(name, ip, prefix, vpc, bouncers, vni)
+    def create_net(self, name, ip, prefix, vpc, vni, bouncers=1):
+        self.api.create_net(name, ip, prefix, vpc, vni, bouncers)
+
+    def get_vpc(self, name):
+        vpc = self.api.get_vpc(name)
+        while vpc["status"] != "Provisioned":
```

Comment:
maybe put "Provisioned" in a variable to prevent duplication (also seen on line #65)